runtime/serde/serde-benchmarks/jvm/src/aws/smithy/kotlin/serde/benchmarks/json/CitmBenchmark.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package aws.smithy.kotlin.serde.benchmarks.json
+
+import aws.smithy.kotlin.runtime.serde.json.JsonToken
+import aws.smithy.kotlin.runtime.serde.json.jsonStreamReader
+import kotlinx.benchmark.*
+import kotlinx.coroutines.runBlocking
+
+@Warmup(iterations = 7, time = 1)
+@Measurement(iterations = 5, time = 1, timeUnit = BenchmarkTimeUnit.MILLISECONDS)
+@OutputTimeUnit(BenchmarkTimeUnit.MILLISECONDS)
+@BenchmarkMode(Mode.AverageTime)
+@State(Scope.Benchmark)
+open class CitmBenchmark {
```

Comment:
**Style**: These annotations are repeated in both benchmark classes and presumably would be on future benchmarks as well. Can we abstract a base class for them? Alternatively, can we define `7`, `5`, `1`, etc. as constants?

Reply:
I think we can define it in the gradle settings as the defaults, will take a look. I don't think constants are going to help, choosing these numbers is likely not going to always be appropriate for every test. The number of warmups, benchmark mode, etc.